uber/templates/registration/form.html

```diff
@@ -5,18 +5,19 @@
 {% include "registration/menu.html" %}
 
 <script type="text/javascript">
-    function isPreassignedBadgeType(badge_type) {
-        return $.inArray(parseInt(badge_type), [{{ STAFF_BADGE }}, {{ SUPPORTER_BADGE }}]) >= 0;
-    }
-    
-    function setGroupLink() {
-        var group = $.val("group_opt");
-        if( group == "" )
-            $("#group_link").hide();
-        else
-            $("#group_link").attr("href", "../groups/form?id=" + group).show();
-    }
-    
+    var isPreassignedBadgeType = function (badge_type) {
```

Comment:
Son of a... yeah, totally.  Especially since we have a ``c.PREASSIGNED_BADGE_TYPES`` config option already!  This is in green but I don't remember adding this during this current refactor, so it's probably an older function that I just cut and pasted to a different place in the file or something.  But yeah, that needs to be fixed, good catch!

I've just filed a ticket for it: https://github.com/magfest/ubersystem/issues/1102